lib/poetry-lock/pyproject.toml

```diff
@@ -28,15 +28,15 @@ networkx = "3.2.1"
 ipython = "8.18.1"
 
 # Package versions from being yanked
-zope-interface = ">=6.4.post0"
+zope-interface = "7.1.0"
 matplotlib = "3.9.0"
 
 # Package versions specified because of github vulnerability reports
 # aiohttp = ">=3.9.5"
 #requests = ">=2.32.0"
 scrapy = ">=2.11.2"
 jinja2 = ">=3.1.4"
-idna = ">=3.7"
+idna = "3.10"
```

Comment:
Same comment on >= here.
Was this updated here because of a vulnerability issue?  If we need 3.10 for code reason it should be in that repo and removed here.

Reply:
Removed.